cmd/volume_test.go

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/mikelangelo-project/capstan/hypervisor"
+	"github.com/mikelangelo-project/capstan/util"
+
+	. "github.com/mikelangelo-project/capstan/testing"
+	. "gopkg.in/check.v1"
+)
+
+type volumesSuite struct {
+	capstanBinary string
+	packageDir    string
+	packageFiles  map[string]string
+	repo          *util.Repo
+}
+
+func (s *volumesSuite) SetUpTest(c *C) {
+	s.packageDir = c.MkDir()
+	os.Chmod(s.packageDir, 0777)
+}
+
+var _ = Suite(&volumesSuite{})
+
+func (s *volumesSuite) TestCreateVolume(c *C) {
+	m := []struct {
+		comment      string
+		volume       Volume
+		expectedMeta string
+		expectedInfo string
+	}{
+		{
+			"create qcow2",
```

Comment:
Please make a test case for "default" config - this will allow us to be alerted whenever defaults are changed.

Reply:
Unfortunately the default value is picked on the CLI level and only passed down here to `CreateVolume` function.

Reply:
EDIT: I've refactored the code so that the default value is picked in the `CreateVolume` function and not on the CLI level. This way we can at least test it. And we actually do - I've added a case as you suggested. Thanks!